schema.sql

```diff
@@ -11,6 +11,46 @@ CREATE TABLE animals (
     weight_kg decimal, 
     species text );
 
-TABLESPACE pg_default;
-
 ALTER TABLE animals ADD COLUMN species text;
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
+-- Below are what I did to achieve this :
+/* 
+Modify animals table:
+Make sure that id is set as autoincremented PRIMARY KEY
+Remove column species
+Add column species_id which is a foreign key referencing species table
+Add column owner_id which is a foreign key referencing the owners table
+*/
+
+-- I created a new "animals" table with the structure below :
+CREATE TABLE animals_new (
+    id serial PRIMARY KEY,
+    name text,
+    date_of_birth date,
+    escape_attempts integer,
+    neutered boolean,
+    weight_kg decimal,
+    species_id integer REFERENCES species(id),
+    owner_id integer REFERENCES owners(id)
+);
```

Comment:
Please add the query used to modify the animal table, following the steps in the [project requirements ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)🤗

![image](https://github.com/ishimwepati/vet_clinic_db/assets/78251927/c07ca360-feeb-4314-bd05-863529550744)


This helps to ensure that you have actually run the queries with success. 💪🤓

Reply:
Thanks a lot, Below are all the steps I used :
<img width="731" alt="Screenshot 2023-09-30 at 02 16 50" src="https://github.com/ishimwepati/vet_clinic_db/assets/14818338/412ff7df-74eb-4452-9df0-8f8b0d4aa306">